src/components/modals/RoleModal.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <Modal
-    v-if="modals.role && availableRoles.length"
-    @close="toggleModal('role')"
-  >
+  <Modal v-if="modals.role && availableRoles.length" @close="close()">
```

Comment:
```suggestion
  <Modal v-if="modals.role && availableRoles.length" @close="close">
```
Parenthesis are optional here